src/main/java/org/primeframework/mvc/message/l10n/ResourceBundleMessageProvider.java

```diff
@@ -79,7 +79,8 @@ public String getMessage(String key, Object... values) throws MissingMessageExce
     String message = getOptionalMessage(key, values);
     if (message == null) {
       ActionInvocation actionInvocation = invocationStore.getCurrent();
-      throw new MissingMessageException("Message could not be found for the URI [" + actionInvocation.actionURI + "] and key [" + key + "]");
+      String uri = actionInvocation != null ? actionInvocation.actionURI : null;
```

Comment:
From looking at this and playing w/ it locally, it looks like the source of noise was from NPEs, which this 'solves', right?

Reply:
Correct. It does solve the NPE. 

I suppose we could also consider not logging this message at all if there is no action invocation.